src/Components/ErrorDisplay/ErrorDisplay.css

```diff
@@ -0,0 +1,58 @@
+.error-display
+{
+	align-items: center;
```

Comment:
Can we take out the extra spaces in this file and make sure everything is aligned the same?